library/src/main/java/com/medium/highlightjs/HighlightJsView.kt

```diff
@@ -47,6 +53,7 @@ class HighlightJsView : WebView, FileUtils.Callback {
     var editMode: Boolean = false
     var latestText: String? = null
     var autoLang: Language? = null
+    var eventKeyHandler: ((keyEvent: KeyEvent?) -> Boolean)? = null
```

Comment:
IMO it's not worth sending null keyEvents to the handler as they can't do anything with it. Android probably never send null keyEvents in the first place.